pages/linux/journalctl.md

```diff
@@ -10,6 +10,10 @@
 
 `journalctl -b -1`
 
+- Show all errors from this boot (-p is priority and 3 is for errors):
+
+`journalctl -p 3 -b`
```

Comment:
We can use long argument format to be easier for understand, and the argument option should be in `{{}}`
```suggestion
`journalctl -b --priority={{3}}`
```